spec/requests/case_contacts_spec.rb

```diff
@@ -90,6 +90,11 @@
     end
 
     context 'with invalid parameters' do
+      # Right now, there is no way for users to edit CaseContacts
+      # And this test is failing for a reason we don't understand.
+      #
+      # We have pended it out because we are not worried about
```

Comment:
This test appears to be no longer pended, so we should remove the out of date comment

Reply:
Good catch! Removed!